lib/rules/color-hex-alpha/index.mjs

```diff
@@ -61,13 +62,6 @@ const rule = (primary) => {
 	};
 };
 
-/**
- * @param {import('postcss-value-parser').Node} node
- */
-function isUrlFunction({ type, value }) {
-	return type === 'function' && value === 'url';
```

Comment:
Technically it could be considered a fix.
The lack of lowercasing is just an oversight.
i.e. not worth a changeset